src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.figures;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        StringBuilder stringBuilder = new StringBuilder();
```

Comment:
there is no need to use `StringBuilder` here - additional cost

you can pass the formatted `String` directly into `println()`

**same should be applied to all other methods**